### ISSUE TITLE
Accessibility considerations

### ISSUE THREAD

**Issue by lgutro** [lgutro/hello-world#44]
Is 'Accessibility' a concern for now or the future?   
Are there any legal requirements?

Test tools:
Wave Accessibility (free)
https://www.softwaretestinghelp.com/web-accessibility-testing-tools/
Nibbler.silktide.com  (among many other things, tests accessibility)

What web accessibility testing is:
https://www.softwaretestinghelp.com/what-is-web-accessibility-testing/

Accessibility
Accessibility—making an application usable by a broad group of people, including those with disabilities and special needs—is achieved by following guidelines on accessible web development. Accessibility testing is an integral part of the testing strategy for modern applications.

More info from  'Lessons Learned in Angular Testing':

AngularJS libraries provide a wide variety of UI widgets, but not all of them are accessible. Testing for accessibility demo versions of UI controls would help to identify those that are accessible and avoid rework later on.

Similar to automation challenges, the way Angular applications render web content presents issues for assistive technologies such as screen readers. Because a page doesn’t reload, assistive technologies cannot provide a proper narrative of the actions. Overcoming these issues requires a significant amount of custom coding, programmatic focus management, and other tricks.

Certain library widgets require light custom tweaking, and some require partial or full rewrites to make them accessible. This is always a trial-and-error process. Working closely with developers and testing prototypes ensures much faster delivery of an accessible and stable UI.

One special risk worth mentioning is regression due to changes in the third-party libraries. As AngularJS actively evolves, upgrading the framework may cause issues with the formerly accessible controls.

https://www.stickyminds.com/article/lessons-learned-testing-angular-applications
